images/miq-app/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM carbonin/manageiq-ruby-container
```

Comment:
Yeah, whatever we layer from needs to be RH aligned (Centos), is there a reason for a specific ruby version (2.3.1) for MIQ purposes? Could we live with a different ruby version coming from SCL? 

Need to investigate what is available, in general SCL builds are preferred way since we are guaranteed to have a RHEL build also available internally and maintained accordingly.